config.json

```diff
@@ -32,7 +32,8 @@
             572,
             773,
             641
-        ]
+        ],
+        "is-alloyed-transmuter-enabled": true
```

Comment:
Can you please update the following and merge them before this PR is merged?

Template: https://github.com/osmosis-labs/infrastructure/blob/55a5946330250d38047d26f8c4865474436db9d3/environments/sqs-osmosis-zone/ansible/roles/prepare_local_folder/templates/sqs/config.json.j2

Configs:
- https://github.com/osmosis-labs/infrastructure/blob/main/environments/sqs-osmosis-zone/environments/stage/ansible/vars/sqs.yaml
- https://github.com/osmosis-labs/infrastructure/blob/main/environments/sqs-osmosis-zone/environments/prod/ansible/vars/sqs.yaml
- https://github.com/osmosis-labs/infrastructure/blob/main/environments/sqs-osmosis-zone/environments/testnet/ansible/vars/sqs.yaml